example/France.NewEBM.json

```diff
@@ -33,21 +50,31 @@
             }
         }
     ],
-    "Variables": [
-        {
-            "COMMENT": "do we even need a section for variables?",
-            "Name": "dFat",
-            "Level": 2,
-            "Factor": "fat"
-        }
-    ],
-    "RiskFactors": [
-        {
-            "COMMENT": "do we even need to list risk factors here?",
-            "Name": "BMI",
-            "Level": 3,
-            "Proxy": "",
-            "Range": [13.88, 39.48983]
-        }
-    ]
+    "AgeMeanHeight": {
```

Comment:
What are the corresponding ages? Is it always assume that it is one point per year? Not sure of the implementation details, but it feels very brittle.

Reply:
In this case, age 0 is the 0th index, etcetera. It assumes age data is complete from zero to max age, which *should* be the case -- no point leaving gaps. Maybe this is a bad assumption?

Reply:
I have no idea if that's a bad assumption - it depends what the real data usually looks like. If you're asked to provide the years, then there's no ambiguity.

Reply:
I think the solution here is proper input sanitisation, making sure that ages are defined and complete up to a minimum of max person age defined in the main config file. Missing data should be a bug.

Reply:
I will do sanitation of the variables above in this PR. I think a wider sanitisation effort would need its own PR though.